extensions/lorem-ipsum/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Lorem Ipsum Changelog
 
+## [New Feature] - 2023-01-02
```

Comment:
Change this to todays date 📅

Reply:
Change it to `{PR_MERGE_DATE}` and raycastbot will automatically replace the merged date.

Reply:
🤯